VSharp.Explorer/Explorer.fs

```diff
@@ -110,29 +115,39 @@ type private SVMExplorer(explorationOptions: ExplorationOptions, statistics: SVM
                 None
             else
                 Some options.randomSeed
+
         match mode with
         | AIMode ->
-            match options.aiAgentTrainingOptions with
+            let useGPU =
+                if options.useGPU.IsSome then
```

Comment:
Look at comments for the previous request.

Reply:
Solved